hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -373,6 +375,15 @@ public void setRestoredRegion(boolean restoredRegion) {
     isRestoredRegion = restoredRegion;
   }
 
+  // Handle table latency metrics
+  private MetricsTableRequests metricsTableRequests;
+
+  public void ifTableRequestsMetricsExist(Consumer<MetricsTableRequests> consumer) {
```

Comment:
sorry i wasn't clear... i was wrong here, and I dont think this method actually improves much. can we revert this part too? sorry

Reply:
Sure, let me try.